docker-compose-zkstack-common.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.2'
+networks:
+  zkstack:
+    driver: bridge
+services:
+  geth:
+    image: "matterlabs/geth:latest"
+    ports:
+      - "127.0.0.1:8545:8545"
+      - "127.0.0.1:8546:8546"
+    volumes:
+    - type: bind
+      source: ./volumes/geth
+      target: /var/lib/geth/data
+    networks:
+      - zkstack
+    container_name: geth
+  postgres:
+    image: "postgres:14"
+    container_name: postgres
+    ports:
+      - "127.0.0.1:5432:5432"
+    volumes:
+      - type: bind
+        source: ./volumes/postgres
+        target: /var/lib/postgresql/data
+    environment:
+      # We bind only to 127.0.0.1, so setting insecure password is acceptable here
+      - POSTGRES_PASSWORD=notsecurepassword
```

Comment:
Maybe `POSTGRES_HOST_AUTH_METHOD=trust` then? That's what we use locally.

Reply:
agree

Reply:
```POSTGRES_HOST_AUTH_METHOD=trust``` won't work for containers in separate docker-network. Connections to PG will be not from localhost netwok.